src/components/Person/Person.jsx

```diff
@@ -1 +1,19 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => {
+  const { name, age, isMarried, sex, partnerName } = person;
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">{`My name is ${name}`}</h2>
+
+      {age && <p className="Person__age">{`I am ${age}`}</p>}
+
+      {isMarried ? (
+        <p className="Person__partner">
+          {`${partnerName} is my ${sex === 'f' ? 'husband' : 'wife'}`}
```

Comment:
consider moving sex === 'f' ? 'husband' : 'wife' to a separate const and using here